teachopencadd/000_template/talktorial.ipynb

```diff
@@ -161,29 +161,12 @@
   }
  ],
  "metadata": {
-  "gist": {
-   "data": {
-    "description": "CADDSeminar_2018/CADD_course_talktorial_template.ipynb",
-    "public": false
-   },
-   "id": ""
-  },
   "kernelspec": {
-   "display_name": "teachopencadd",
-   "language": "python",
-   "name": "teachopencadd"
+   "display_name": "",
+   "name": ""
   },
   "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "name": ""
   }
  },
  "nbformat": 4,
```

Comment:
@jaimergp, have you done this manually for all notebooks or is there an automated way to clear metadata from notebooks?

Reply:
There must be a way but in this case I did it manually after figuring out the click pattern :D